src/utils/SetupUtil.js

```diff
@@ -56,34 +66,48 @@ export default {
     return _retryPromise.promise;
   },
 
-  setup() {
-    return util.isLinux() ? this.nativeSetup() : this.nonNativeSetup();
-  },
-
-  async nativeSetup () {
+  async setup () {
     while (true) {
       try {
-        docker.setup('localhost', machine.name());
-        docker.isDockerRunning();
-
-        break;
+        if (util.isNative()) {
+          localStorage.setItem('setting.useNative', true);
```

Comment:
Why store this setting?

If the user starts Kitematic and the socket is available, shouldn't it just use that by default, always? If the socket isn't available, then maybe it should just always use VBox?